docs/contacts/custom_fields.rst

```diff
@@ -105,3 +105,28 @@ If you configure this option, the new Custom Field becomes visible in the list o
 This configuration helps prevent **http** request timeouts because it handles the long-running SQL query to create the new table column as a background task.
 
 To mitigate the table lock issue, run the command only once daily when you know that most of your audience is offline. With less traffic going into Mautic, the chances of encountering a problem are lower.
+
+Locally defined countries and regions
+*************************************
+
+Since Mautic 5.1 it's possible to define custom countries and regions via locally hosted JSON files. This is useful when you have a specific set of countries or regions that you want to use in your Mautic instance. You can define these in a file called ``countries.json`` or ``regions.json`` located in your defined ``upload_dir``. Example code snippets are below:
```

Comment:
```suggestion
Since Mautic 5.1 it's possible to define custom countries and regions via locally hosted JSON files. This is useful when you have a specific set of countries or regions that you want to use in your Mautic instance. You can define these in a file called ``countries.json`` or ``regions.json`` located in your defined ``upload_dir`` (``media/files`` by default). Example code snippets are below:
```